CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `spotlight/SpotlightRootDecorator` to change non touch mode when `enter` key is down in touch mode
```

Comment:
Please write the changelog under packages/spotlight/CHANGELOG.md.

Reply:
```suggestion
- `spotlight/SpotlightRootDecorator` to show focus effect when `enter` key is pressed in touch mode
```

Reply:
Fix it. Thanks~